### DISCUSSION TITLE
Generate profiles not updating class with changed action list

### DISCUSSION THREAD

**Discussion by Childishforlife** [simulationcraft/simc#6600] [General]
I recently made a change to "Github\Childish\simc\engine\class_modules\sc_shaman.cpp", but whenever I run the "generate_profiles.bat" file, the ele shaman profiles are not changed at all.

What do I need to do to enable re-generating the ele shaman profiles?

**Comment by scamille** [comment]
Did you recompile simc.exe that is used inside generate_profiles.bat to regenerate the profile?

If you use the old simc.exe without recompiling it yourself, your changes to a source file will not be reflected.

If that isn't the case then I am a bit at a loss. You could share your changes and we could have a look at it.

**Reply by Childishforlife** [reply]
I think that may be the case, my .exe was from 9.1.5, not 9.2

Will try recompiling and see if that fixes things, thanks!
